ci/hosts

```diff
@@ -9,3 +9,29 @@ localhost ansible_connection=local
 [stockpile]
 # The host stockpile is running from
 localhost ansible_connection=local
+undercloud-0 ansible_host=undercloud_ip ansible_connection=ssh
+
+[stockpile:vars]
+stockpile_user=stack
+stockpile_output_path=/tmp/stockpile.json
```

Comment:
Please change this to be in the jenkins workspace, so we can look at it. However, it is overwritten each time you call featureset001.yml with certain tags in a loop. Is there a way we can change the stockpile_output_path for each iteration of running featureset001.yml as in lines 30-31 of osp_ci_test.sh?
Thoughts? Do you think that is useful?

Reply:
@sai I have used the default path for stockpile_output_path https://github.com/cloud-bulldozer/stockpile/blob/master/group_vars/all.yml#L6 but we can change it into Jenkins workspace if needed. Also, why do we need to change the stockpile_output_path for each iteration of running featureset001.yml ? as for each iteration it stores the output of the featureset001.yml after running

Reply:
Isn't the output always re-written after running the playbook by dump-facts role, since u run the playbook multiple times, with a different tag each time?

Reply:
yes, it gets overwritten but for each iteration, it checks if the json generated in  stockpile_output_path  is valid or not and then can be viewed in the results_osp.markdown

Reply:
Yes, But I'm saying it would be nice to save the raw output for each run of the playbook for each tag.

Reply:
Yeah but for each iteration, if we store the output in file then will have to many files created

Reply:
I understand, thats OK :)